ls1intum/Artemis
# Bugfix: Correct highlighting of selected tour element

### Checklist

### Motivation and Context
This PR should solve the following issue: https://github.com/ls1intum/Artemis/issues/1093

### Description
This PR observes alerts while the guided tour service is active and changes the position of the highlight element if it has been changed due to the display of the alert.

### Steps for Testing
1. Go the the Tutorial Course
2. Go the the modeling exercise and submit a UML model
3. Right after the alert is displayed, start the tour and navigate to a step with highlight element
4. Wait until the alert is not displayed anymore -> the position of the highlight element should be correct

### Screenshots
![12](https://user-images.githubusercontent.com/33764166/71733064-f91a6100-2e48-11ea-948a-0e8c577de02e.gif)
